gravitee-policy-quota/src/main/java/io/gravitee/policy/quota/QuotaPolicy.java

```diff
@@ -156,36 +170,62 @@ public void onError(Throwable e) {
                 });
     }
 
-    private String createRateLimitKey(Request request, ExecutionContext executionContext) {
-        // Rate limit key must contain :
-        // 1_ PLAN_ID
-        // 1_ SUBSCRIPTION_ID
-        // 2_ Rate Type (rate-limit / quota)
-        // 3_ RESOLVED_PATH (policy attached to a path rather than a plan)
+    private long evaluateActualLimit(ExecutionContext executionContext, QuotaConfiguration quotaConfiguration) {
+        if (limitEvaluationMethods.get(quotaConfiguration) == LimitEvaluationMethod.PARSE_LONG) {
```

Comment:
```java
limitEvaluationMethods.getOrDefault(quotaConfiguration, LimitEvaluationMethod.PARSE_LONG)
```

Reply:
Sorry, my Java is kind of rusty... I see the request is merged so I assume you fixed it in master :wink:

Reply:
No worries, I did it by myself after the merge :)